internal/extsvc/codehost.go

```diff
@@ -74,7 +74,12 @@ func NormalizeBaseURL(baseURL *url.URL) *url.URL {
 // matching codehost from the given list.
 func CodeHostOf(name api.RepoName, codehosts ...*CodeHost) *CodeHost {
 	repoNameParts := strings.Split(string(name), "/")
```

Comment:
I think this should do the trick. We only need 2 parts.
```suggestion
	repoNameParts := strings.SplitN(string(name), "/", 2)
```

Reply:
Wouldn't that mean repo name of the format - `github.com/foo` will pass as valid repo names and then throw the same error that we get right now? :thinking:

Reply:
The problem we have right now is that the hostname matches but shouldn't, not the path.

Reply:
Fair enough. Fixed and updated docs.